kpTranslator/config.py

```diff
@@ -2,10 +2,15 @@
 import connexion
 from flask_sqlalchemy import SQLAlchemy
 from flask_marshmallow import Marshmallow
+from kpTranslator import encoder
 
 basedir = os.path.abspath(os.path.dirname(__file__))
+db_file = os.path.join(basedir, 'translator_kg.db')
 
-from kpTranslator import encoder
+# Delete database file if it exists currently
```

Comment:
This should go in a script in the database file or else your database will be deleted each time the application starts.  You don't want that in production and it won't work when you use another kind of database other than sqlite.

Reply:
Good catch. Thanks. I've moved this database deletion code to build_database.py. I'll be interested in recommendations once we switch to a different kind of database for production. For now, I'm using sqlite for convenience while developing the query processing/response code.

Reply:
Thanks for catching this. Moved this code to build_database.py script.